pkg/mounter/fake_safe_mounter.go

```diff
@@ -33,6 +34,10 @@ type FakeSafeMounter struct {
 
 // NewFakeSafeMounter creates a mount.SafeFormatAndMount instance suitable for use in unit tests.
 func NewFakeSafeMounter() (*mount.SafeFormatAndMount, error) {
+	if runtime.GOOS == "windows" {
+		return NewSafeMounter(true)
```

Comment:
With this change, we'll need to re-enable CSI Proxy in the Windows unit test Github actions. I removed the dependency earlier because I thought it unnecessary, but Andy sees some value in continuing to rely on CSI Proxy for the unit tests. I've been meaning to add it back in, and this will do it for us. To enable CSI Proxy in the unit tests, and the following to the `run` section before `go test...`:

```sh
# start the CSI Proxy before running tests on windows
Start-Job -Name CSIProxy -ScriptBlock {
Invoke-WebRequest https://github.com/andyzhangx/demo/raw/master/windows/csi-proxy-v1.0.0-rc.1.tar.gz -OutFile csi-proxy.tar.gz;
tar -xvf csi-proxy.tar.gz
.\bin\csi-proxy.exe --kubelet-path $pwd
};
Start-Sleep -Seconds 30;
Write-Output "getting named pipes"
[System.IO.Directory]::GetFiles("\\.\\pipe\\")
```

Reply:
I believe this addition will need to be made to the `.github/actions/windows-v2.yaml` file as well.

Reply:
Make sense.